src/_scss/pages/homepage/hero/_covid-hero.scss

```
@@ -38,107 +84,96 @@ $dot-color: #dcefef;
-                              font-size: rem(40);
-                            }
-                            font-weight: $font-bold;
-                        }
-                        &:first-of-type {
-                          font-size: rem(24);
-                            @media(min-width: $medium-screen) {
-                              font-size: rem(30);
-                            }
-                          font-weight: $font-light;
-                        }
-                        &:nth-of-type(2) {
-                          font-size: rem(30);
-                            @media(min-width: $medium-screen) {
-                              font-size: rem(40);
-                            }
-                        }
-                        &:nth-of-type(3) {
-                          font-size: rem(30);
-                          @media(min-width: $medium-screen) {
-                            font-size: rem(40);
-                          }
-                        }
-                        &:nth-of-type(4) {
-                          width: auto;
-                          strong {
-                            font-size: rem(30);
-                              @media(min-width: $medium-screen) {
-                                font-size: rem(40);
-                              }
-                          }
-                        }
+                    font-weight: $font-semibold;
+                  }
+                  strong {
+                    font-size: rem(30);
+                    @media(min-width: 1027px) {
+                      font-size: rem(40);
                     }
-                    @import "../../../components/tooltips/index";
-                      .tooltip-wrapper {
-                        display: inline-block;
-                        z-index: 9;
-                        .tooltip__hover-wrapper {
+                    font-weight: $font-semibold;
+                  }
+                  @import "../../../components/tooltips/index";
+                    .tooltip-wrapper {
+                      display: inline-block;
+                      z-index: 9;
+                      .tooltip__hover-wrapper {
+                        svg {
+                          color: $color-covid-19-obligations;
+                        }
+                        &:hover {
                           svg {
-                            color: $color-covid-19-obligations;
-                          }
-                          &:hover {
-                            svg {
-                              color: $color-covid-dark-purple;
-                            }
+                            color: $color-covid-dark-purple;
                           }
                         }
-                    }
+                      }
+                  }
                 }
                 p {
-                    color: white;
+                    color: #303030;
```

This is pretty similar to `$color-gray-dark` (we have a ticket to meet with design and ensure our color variables are consistent with what they use in mocks, and it will be easier to update all instances with a variable)
```suggestion
                    color: $color-gray-dark;
```